poetry.lock

```diff
@@ -1,10 +1,9 @@
-# This file is automatically @generated by Poetry 1.4.0 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
 
 [[package]]
 name = "annotated-types"
 version = "0.6.0"
 description = "Reusable constraint types to use with typing.Annotated"
-category = "main"
```

Comment:
@samj1912 did you intentionally add these categories with poetry 1.4.0 in #916?

Reply:
Nope, just did a poetry update